packages/workflow-app-textedit/Readme.md

```diff
@@ -0,0 +1,33 @@
+# TextEdit workflow app
+
+## Installation
+
+```
+npm i workflow-app-textedit
+```
+
+## Usage
+
+```
+import TextEdit from 'workflow-app-textedit';
+
+export default {
+  name: 'workflow-example',
+  type: 'workspace',
+  children: [
+    {
+      type: 'layout',
+      layout: 'splith',
+      percent: 1.0,
+      children: [
+        {
+          ...TextEdit,
+          cwd: __dirname,
+          filename: 'index.js',
+          percent: 0.5,
```

Comment:
Note: there are a few corners of workflow which I need some more thought. I'll address them in you PRs as the show up, so that we have the context to discuss them. Most of these will be needing their own issues. :) Some of my comments, like the following will not have more questions than answers.

This layout is a bit problematic. You are not able to specify which half of the screen you would like to cover. I think this will run on each platform and cover the left side (except on i3 where it will cover the whole screen), but there is no way to make it cover the right. And I do not think we want to add a way to do so. This layout can be achieved with `workflow-layout-yoga` where it makes perfect sense, you have the flexibility to float to the right and i3 would do the right thing. 

So we pretty much have two options here. Fail fast and complain about the empty space in the split node, or make some sort of dummy node for the open area, this node could be interested by default in the rightmost or bottommost space, and be a node type you could add yourself to the left or top. This could be done with an empty app node, but I'm not sure that is the right thing.we could also make this node grow to the size of the parent, like i3 does.

Reply:
This value should probably have been 1 when it was commited. I just changed the value to 0.5 to check that it didn't always fill the whole screen and forgot about it.

Reply:
Yeah, it stated out like it... :P